app/assets/templates/header.html

```diff
@@ -11,7 +11,7 @@
           </button>
           <ul class="dropdown-menu list-inline" role="menu">
             <li class="dropdown-list" ng-repeat='category in categories'>
-              <a ng-href="#{{category.link}}" class="dropdown-category">{{ category.name }}</a>
+              <a ng-href="#/categories/{{ category.link }}" class="dropdown-category">{{ category.name }}</a>
```

Comment:
If you're sure that you won't get confused it's fine. however in the long term such discrepancies can lead to errors! How long would it take to standardise?